Add category tags for tutorials

bebatut opened the pull request:
As asked in #349, this PR should add the possibility to have tags for the tutorials in the metadata files that will be display below the tutorial name on the topic page: 

![screen shot 2018-01-30 at 15 15 10](https://user-images.githubusercontent.com/1842467/35570820-96872b06-05d0-11e8-9181-c402b2c2acdf.png)

Several tags can be added for a tutorial as a list:

![screen shot 2018-01-30 at 15 19 57](https://user-images.githubusercontent.com/1842467/35570983-1c53791a-05d1-11e8-89c1-cb29b7b3d23e.png)

PS: Currently the labels are not links. But we could think (in the future) to extend this feature to list all the tutorials with the same tag. But I am not sure if it will be easy...

bgruening commented on topics/transcriptomics/metadata.yaml:
```diff
@@ -57,6 +57,8 @@ material:
     hands_on: yes
     slides: no
     workflows: yes
+    tags:
```
Most of our tutorials are for eukaryotes and I'm not sure its a good idea to add this particular label to all our tutorials.
I added one for prokaryotes, which imho makes more sense.

bebatut commented:
@bgruening I added more "prokaryote" and removed the "eukaryote" tag
I see also the issue with table

![screen shot 2018-04-27 at 12 13 14](https://user-images.githubusercontent.com/1842467/39357985-d2ab15ba-4a14-11e8-81d5-bcc2ee8c0e9c.png)

@willdurand any idea how we could fix that?

willdurand commented:
> @willdurand any idea how we could fix that?

we can center the icons (vertically) but that's it. You would have the same output if the titles were longer, I don't think it is an issue.

bgruening commented:
Thanks B!
